cpp/modmesh/view/R3DWidget.cpp

```diff
@@ -99,23 +100,6 @@ void R3DWidget::resizeEvent(QResizeEvent * event)
     m_container->resize(event->size());
 }
 
-void R3DWidget::resetCamera() const
-{
-    Qt3DRender::QCamera * camera = m_view->camera();
-
-    // Set up the camera.
-    camera->lens()->setPerspectiveProjection(45.0f, 16.0f / 9.0f, 0.1f, 1000.0f);
-    camera->setPosition(QVector3D(0.0f, 0.0f, 10.0f));
-    camera->setViewCenter(QVector3D(0.0f, 0.0f, 0.0f));
-    camera->setUpVector(QVector3D(0.f, 1.f, 0.f));
-
-    // Set up the camera control.
-    auto * control = m_scene->controller();
-    control->setCamera(camera);
-    control->setLinearSpeed(50.0f);
-    control->setLookSpeed(180.0f);
-}
-
```

Comment:
resetCamera moved to CameraController

Reply:
Moving it to `CameraController` makes it more testable.  I like it.